package com.sequoiacm.cloud.adminserver.exception;

public final class StatisticsError {
    public static final String RECORD_NOT_EXISTS = "record_not_exists";
    public static final String RECORD_EXISTS = "record_exists";
    public static final String WORKSPACE_NOT_EXISTS = "workspace_not_exists";
    public static final String INVALID_ARGUMENT = "invalid_argument";
    public static final String MISSING_ARGUMENT = "missing_argument";
    public static final String INTERNAL_ERROR = "internal_error";
    public static final String BAD_REQUEST = "bad_request";
    public static final String METASOURCE_ERROR = "metasource_error";
    public static final String LOCK_ERROR = "lock_error";
    public static final String QUOTA_ALREADY_ENABLE = "quota_already_enable";
    public static final String QUOTA_NOT_ENABLE = "quota_not_enable";
    public static final String QUOTA_ALREADY_DISABLE = "quota_already_disable";
    public static final String QUOTA_SYNCING = "quota_syncing";
    public static final String QUOTA_SYNC_TIME_DIFF_TOO_LARGE = "quota_sync_time_diff_too_large";
    public static final String UNAUTHORIZED_OPERATION = "Unauthorized operation";
}
